api_docs/changelog.md

```diff
@@ -20,6 +20,19 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 284**
+
+* [`GET /events`](/api/get-events): An event with `type: "saved_replies"` is
+  sent to the current user when a saved reply is created or deleted.
+* [`POST /register`](/api/register-queue): Added a new `saved_replies` field
+  for current user's saved replies.
```

Comment:
Generally, we list `events` and `register` together on a single line; probably you should follow that convention.

Reply:
Changed it to --
```md
* [`GET /events`](/api/get-events), [`POST /register`](/api/register-queue):
  An event with `type: "saved_replies"` is sent to the current user when a
  saved reply is created or deleted.
```